client/landing/jetpack-cloud/components/fix-all-threats-dialog/index.tsx

```diff
@@ -115,6 +116,14 @@ const FixAllThreatsDialog = ( {
 								Go back
 							</Button>
 						) }
+						{ firstStep === currentStep && (
+							<Button
+								className="fix-all-threats-dialog__btn fix-all-threats-dialog__btn--cancel"
+								onClick={ onCloseDialog }
+							>
+								{ translate( 'Cancel' ) }
+							</Button>
+						) }
 						<Button primary className="fix-all-threats-dialog__btn" onClick={ onConfirmation }>
 							Fix all threats
```

Comment:
We should probably also translate this. 
```suggestion
							{ translate( 'Fix all threats' ) }
```